library/server.c

```diff
@@ -219,6 +219,8 @@ void dnet_server_node_destroy(struct dnet_node *n)
 {
 	dnet_log(n, DNET_LOG_DEBUG, "Destroying server node.");
 
+	dnet_node_stop_common_resources(n);
+
```

Comment:
This is a contradictory to what is written in comments about monitor below.
Please update both monitor comments and add description of the new cleanup logic.